tutorials/math/random_number_generation.rst

```diff
@@ -186,20 +186,24 @@ and ``to``, and returns a random integer between ``from`` and ``to``:
 Get a random array element
 --------------------------
 
-We can use random integer generation to get a random element from an array:
+We can use random integer generation to get a random element from an array,
+or use the :ref:`Array.pick_random<class_Array_method_pick_random>` method
+to do it for us:
 
 .. tabs::
  .. code-tab:: gdscript GDScript
 
     var _fruits = ["apple", "orange", "pear", "banana"]
 
     func _ready():
-        randomize()
-
         for i in range(100):
             # Pick 100 fruits randomly.
             print(get_fruit())
 
+        for i in range(100):
+            # Pick 100 fruits randomly, this time using the `Array.pick_random()`
+            # helper method. This has the same behavior as `get_fruit()`.
+            print(_fruits.pick_random())
```

Comment:
`pick_random` is also available in C# (as `PickRandom` in `Godot.Collections.Array`) so you could add this to the C# code, but you'll have to change the the type of the `_fruits` field like so:

```csharp
private Godot.Collections.Array<string> _fruits = new Godot.Collections.Array<string> { "apple", "orange", "pear", "banana" };
```

We usually prefer the BCL collections which is probably why this example was using `string[]` but I guess it makes sense to use Godot collections in this case, in order to document this API.